src/CarbonAware.WebApi/test/integrationTests/UnconfiguredWebApiTests.cs

```diff
@@ -31,4 +31,15 @@ public async Task FakeEndPoint_ReturnsNotFound()
         Assert.That(result, Is.Not.Null);
         Assert.That(result.StatusCode, Is.EqualTo(HttpStatusCode.NotFound));
     }
+
+    [Test]
+    public async Task Telemetry_HealthCheck_ReturnsOK()
+    {
+        Environment.SetEnvironmentVariable("CarbonAwareVars__TelemetryProvider", "ApplicationInsights");
```

Comment:
I don't think you need to set this env vars to check for the health of the system.

Reply:
I added this to increase integration tests coverage.

Reply:
> I added this to increase integration tests coverage.

Got it. It feels confusing, since these variables don't do anything related to health check. Perhaps create a new test that basically verifies that those env vars are loaded, to make it the coverage 'happy'

Reply:
I deleted the test. Feels like it can be better tested in Unit tests, like test that AddApplicationInsigtsTelemetry method is actually called on the mocked services. Sent message to Kristi about this.

Reply:
Test case in unit tests - https://statics.teams.cdn.office.net/evergreen-assets/safelinks/1/atp-safelinks.html